ehrapy/api/_anndata_util.py

```diff
@@ -36,3 +36,73 @@ def get_column_values(adata: AnnData, indices: Union[int, List[int]]) -> np.ndar
         :class:`~numpy.ndarray` object containing the column values
     """
     return np.take(adata.X, indices, axis=1)
+
+
+def assert_encoded(adata: AnnData):
```

Comment:
This method will eventually overlap with https://github.com/theislab/ehrapy/issues/182

But let's keep it for now!